torch/onnx/symbolic.py

```diff
@@ -852,3 +857,62 @@ def symbolic(g, input, all_weights, h0, batch_sizes):
         return prev_output, h_outs
 
     return symbolic
+
+
+def rnn_trace_override_symbolic(cell_type, func, sym, g, input, weights, hiddens, batch_sizes):
```

Comment:
Some extra comments would be nice. E.g. what `func` is, what's the format of `weights`, `hiddens`, what are their types?